core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -217,14 +217,23 @@ describe('Permissions for private explorations', function() {
     explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
   });
-  it('should be correct for collaborators', function() {
+  it('should not allow adding users if title is not given', function() {
```

Comment:
Instead of "adding users" I think we can be specific about what we are doing, like "not allow adding roles if the exploration title ..." or  "not allow adding collaborator if the exploration title ..."